bugrap/src/main/java/org/vaadin/bugrap/BugrapUI.java

```diff
@@ -23,23 +24,33 @@
  */
 @Theme("valo")
 public class BugrapUI extends UI {
-    @Override
-    protected void init(VaadinRequest vaadinRequest) {
-    	
-    		Navigator navigator = new Navigator(this,  this);
-    		
-        final LoginView loginView = new LoginView();
-        loginView.setModel(new LoginModel(navigator));
-        navigator.addView(BaseModel.NAV_LOGIN, loginView);
-    	
-    		final ReportsView reportsView = new ReportsView();
-    		reportsView.setModel(new ReportsModel(navigator));
-    		navigator.addView(BaseModel.NAV_REPORT, reportsView);
-    		navigator.navigateTo(BaseModel.NAV_LOGIN);
-    }
+    
+	@Override
+	protected void init(VaadinRequest vaadinRequest) {
+		Navigator navigator = new Navigator(this, this);
 
-    @WebServlet(urlPatterns = "/*", name = "MyUIServlet", asyncSupported = true)
+		final LoginView loginView = new LoginView();
+		loginView.setModel(new LoginModel(navigator));
+		navigator.addView(BaseModel.NAV_LOGIN, loginView);
+
+		final ReportsView reportsView = new ReportsView();
+		reportsView.setModel(new ReportsModel(navigator));
+		navigator.addView(BaseModel.NAV_REPORT, reportsView);
+		navigator.navigateTo(BaseModel.NAV_LOGIN);
+	}
+
+    @WebServlet(urlPatterns = "/*", name = "BugrapUIServlet", asyncSupported = true)
     @VaadinServletConfiguration(ui = BugrapUI.class, productionMode = false)
-    public static class MyUIServlet extends VaadinServlet {
+    public static class BugrapUIServlet extends VaadinServlet {
+    	
+    		@Override
+    		protected void servletInitialized() throws ServletException {
+    			super.servletInitialized();
+    			if (DatabaseHelper.initializeIfEmpty(BaseModel.getRepository())) {
+    				System.out.println("Database was empty, but initialized. Reautenticating...");
```

Comment:
Instead of System.out.println use [Java's Logger](https://docs.oracle.com/javase/7/docs/api/java/util/logging/Logger.html).